src/components/HURUmap/KeyMetric/index.js

```diff
@@ -0,0 +1,58 @@
+import { Typography, LinearProgress } from "@material-ui/core";
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import React from "react";
+
+import useStyles from "./useStyles";
+
+const KeyMetric = ({ formattedValue, value, title, color, description }) => {
+  const classes = useStyles();
+
+  if (!(formattedValue && title)) {
```

Comment:
```suggestion
  if (!(value && title)) {
```

`formattedValue` is option; we use it when available

Reply:
Got it. Fixed